crates/uv-python/src/pe_version.rs

```diff
@@ -0,0 +1,145 @@
+//! Extract version information from Windows PE executables.
+use std::path::Path;
+
+#[cfg(target_os = "windows")]
```

Comment:
Maybe we can make this whole module cfg'd for windows like `microsoft_store ` is and drop the `try_extract_version_from_pe` for non-windows?